wcivf/templates/base.html

```diff
@@ -71,7 +71,12 @@
             </aside>
         </form>
     {% endif %}
+
 {% endblock base_language_menu %}
+{% get_current_language as LANGUAGE_CODE %}
+{% if LANGUAGE_CODE == 'cy' %}
```

Comment:
I can't seem to implement this the way it's suggested in the design system. The subheader site title shows in both English and Welsh versions but I only want it to appear in the latter.

Reply:
@GeoWill I tried to follow what you have done for WDIV, but I'm not clear on how this works.

Reply:
How is it not working? 
Have you tried printing `LANGUAGE_CODE` (or inspecting in debug toolbar) to check what it's coming out as - i.e. is it always returning `en`?

Reply:
The conditional isn't working, it's always on display in Welsh as shown here ![image](https://github.com/DemocracyClub/WhoCanIVoteFor/assets/7017118/6ab4fbc4-a0f2-4275-b6e2-7a3a75611113)

Reply:
OK. So what are the results of the things I suggested?

Reply:
The value of `LANGUAGE_CODE` prints as `cy`

Reply:
Huh. I checked it out and had a look. I think this is what you want: 
```
{% block language_code %}
    {% get_current_language as LANGUAGE_CODE %}
    {% if LANGUAGE_CODE == 'cy' %}
        <em lang="cy">Pwy alla i bleidleisio ar gyfer?</em>
    {% endif %}
{% endblock language_code %}
```
 I don't really understand why, but I guess something to do with template scopes.

Reply:
There's no subtitle when it's English though. Is that what you want?

Reply:
I figured it out. The conditional needed to go inside the block tags. Thanks.

Reply:
Oh yes, you did too :)